src/plugins/timeConductor/TimePopupFixed.vue

```diff
@@ -219,18 +219,21 @@ export default {
         return false;
       }
     },
-    submit() {
+    handleFormSubmission(shouldDismiss) {
+      // Validate bounds before submission
       this.validateAllBounds('startDate');
       this.validateAllBounds('endDate');
-      this.submitForm(!this.isDisabled);
-    },
-    submitForm(dismiss) {
-      // Allow Vue model to catch up to user input.
-      // Submitting form will cause validation messages to display (but only if triggered by button click)
-      this.$nextTick(() => this.setBoundsFromView(dismiss));
+
+      // Submit the form if it's valid
+      if (!this.isDisabled) {
+        this.setBoundsFromView(shouldDismiss);
```

Comment:
let's see if we can add coverage for this in 3 minutes of investigation. Ignore otherwise